tests/pcie/test_pcie_device.cpp

```diff
@@ -0,0 +1,52 @@
+
```

Comment:
Thanks for the test! 
Can we make this run on CI? Is this currently being bundled in a separate binary during build?

Reply:
It is a separate binary, since it can run independent of the architecture.  I've added to `run-tests.yml` - please take a look.

Reply:
Looks like I also needed to add the test to the CMake target `umd_tests` for CI to pick it up.